cypress/integration/linkingAccessionIds.spec.ts

```diff
@@ -1,255 +0,0 @@
-describe("Linking Accession Ids", function () {
```

Comment:
if we don't test this any more does this mean we removed this feature ?

Reply:
I was thinking that refactored accordion flow test in `app.spec.ts` which uses linked accession ID's should be sufficient in this case. Maybe we could use Cypress aliases there to match the accession ID's.